cross/cloudflared/Makefile

```diff
@@ -14,10 +14,17 @@ LICENSE  = Apache-2.0
 
 GO_SRC_DIR = $(WORK_DIR)/$(PKG_NAME)-$(PKG_VERS)
 GO_BIN_DIR = $(GO_SRC_DIR)/$(PKG_NAME)
-COMPILE_TARGET = cloudflared_compile
+COMPILE_TARGET = install-go cloudflared_compile
 
 include ../../mk/spksrc.cross-go.mk
 
 .PHONY: cloudflared_compile
 cloudflared_compile:
-	@$(RUN) $(MAKE) VERSION=$(PKG_VERS) LINK_FLAGS="-s -w" cloudflared
+	@$(RUN) $(MAKE) PATH="/tmp/go/bin:${PATH}" VERSION=$(PKG_VERS) LINK_FLAGS="-s -w" cloudflared
+
+.PHONY: install-go
+install-go:
+ifeq (,$(wildcard /tmp/go))
+	$(RUN) GOARCH= GOOS= GOROOT_BOOTSTRAP=$(WORK_DIR)/../../../native/go/work-native/go/ \
+	$(WORK_DIR)/$(PKG_NAME)-$(PKG_VERS)/.teamcity/install-cloudflare-go.sh
+endif
```

Comment:
Please avoid to define more than one target for `COMPILE_TARGET`

There is no need for an extra target `install-go`.
The compile target could be written like this:
```
.PHONY: cloudflared_compile
cloudflared_compile:
ifeq (,$(wildcard /tmp/go))
        @$(MSG) "   Call install-cloudflare-go.sh"
	@$(RUN) GOARCH= GOOS= GOROOT_BOOTSTRAP=$(WORK_DIR)/../../../native/go/work-native/go/  \
        .teamcity/install-cloudflare-go.sh
endif
        @$(MSG) "   Build cloudflared"
	@$(RUN) $(MAKE) PATH="/tmp/go/bin:${PATH}" VERSION=$(PKG_VERS) LINK_FLAGS="-s -w" cloudflared
```

Remarks:
- merged both targets into one
- don't forget to remove `install-go` from `COMPILE_TARGET`
- added `$(MSG)` messges to print out the two steps
- removed `$(WORK_DIR)/$(PKG_NAME)-$(PKG_VERS)/` for calling `install-cloudflare-go.sh` - the $(RUN) macro does a `cd $(WORK_DIR)/$(PKG_DIR)` i.e. `cd $(WORK_DIR)/$(PKG_NAME)-$(PKG_VERS)` so the full path is not required.
- added `@` to `$(RUN)` to avoid printing the whole env to the console

-OR-

If you prefer to have two dedicated targets, you should define `PRE_COMPILE_TARGET` and `COMPILE_TARGET` and rename `install-go` to `cloudflared_pre_compile`

i.e.
```
PRE_COMPILE_TARGET = cloudflared_pre_compile
COMPILE_TARGET = cloudflared_compile
```

Reply:
Done